composer.json

```diff
@@ -9,10 +9,10 @@
   },
   "authors": [{
     "name": "Christian Bach",
-    "url": "http://tablesorter.com/"
+    "homepage": "http://tablesorter.com/"
```

Comment:
Thanks for updating this! Can you completely remove this `homepage` entry because the page no longer exists :(